x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/charts/anomaly_chart/anomaly_chart.tsx

```diff
@@ -43,7 +43,7 @@ export const AnomalyChart: FC<Props> = ({
   const data = chartType === CHART_TYPE.SCATTER ? flattenData(chartData) : chartData;
   const xDomain = getXRange(data);
   return (
-    <div style={{ width, height }} data-test-subj="mlAnomalyChart">
+    <div style={{ width, height }} data-test-subj={`mlAnomalyChart ${CHART_TYPE[chartType]}`}>
```

Comment:
I know that @dmlemeshko is changing the way that test subjects with spaces work, so I want to make sure that this and that PR don't collide for disaster. Can you two coordinate to make sure that your PRs get merged in the right order?